pages/blog/event-driven-api-documentation-made-simple-clientside-rendering.md

```diff
@@ -0,0 +1,256 @@
+  schema='{"asyncapi":"2.4.0","info":{"title":"Account Service","version":"1.0.0","description":"This service is in charge of processing user signups"},"channels":{"user/signedup":{"subscribe":{"message":{"$ref":"#/components/messages/UserSignedUp"}}}},"components":{"messages":{"UserSignedUp":{"payload":{"type":"object","properties":{"displayName":{"type":"string","description":"Name of the user"},"email":{"type":"string","format":"email","description":"Email of the user"}}}}}}}'
+
+  config='{"show": {"sidebar": false}}'
+
+  cssImportPath="https://unpkg.com/@asyncapi/react-component@1.0.0-next.39/styles/default.min.css">
+</asyncapi-component>
+```
+
+2️⃣ If you need support for old browsers then you need to add this script as well
+```html
+<script src="https://unpkg.com/@webcomponents/webcomponentsjs@2.5.0/webcomponents-bundle.js"></script>
+```
+
+That is it! 🤯 Just awesome!
+
+---
+
+### <a name="sb"></a>Standalone Bundle
+
+If you want to render documentation from your AsyncAPI file without the use of any framework but with just an HTML webpage then you will need the [Standalone bundle](https://github.com/asyncapi/asyncapi-react/blob/next/docs/usage/standalone-bundle.md).
+
+1️⃣ All you need is just a basic HTML template.
+
+2️⃣ To get AsyncAPI document styling, in the **head** element enter:
+```html
+<link rel="stylesheet" href="https://unpkg.com/@asyncapi/react-component@1.0.0-next.39/styles/default.min.css">
+```
+
+3️⃣ In the **body** element type the following:
+
+```html
+ <div id="asyncapi"></div>
+
+    <script src="https://unpkg.com/@asyncapi/react-component@1.0.0-next.39/browser/standalone/index.js"></script>
+    <script>
+        AsyncApiStandalone.render({
+            schema: '{"asyncapi":"2.4.0","info":{"title":"Account Service","version":"1.0.0","description":"This service is in charge of processing user signups"},"channels":{"user/signedup":{"subscribe":{"message":{"$ref":"#/components/messages/UserSignedUp"}}}},"components":{"messages":{"UserSignedUp":{"payload":{"type":"object","properties":{"displayName":{"type":"string","description":"Name of the user"},"email":{"type":"string","format":"email","description":"Email of the user"}}}}}}}'
+            ,
+            config: {
+                show: {
+                    sidebar: true,
+                }
+            },
+        }, document.getElementById('asyncapi'));
+    </script>
+```
+
+This fetches everything required from the bundle
+```html
+<script src="https://unpkg.com/@asyncapi/react-component@1.0.0-next.39/browser/standalone/index.js"></script>
+```
+
+There is one more way to configure the AsyncAPI component.
+You can do it through **config** props; the same as for the normal React component.
+
+My **Web Component** and **Standalone Bundle** usage examples have `config='{"show": {"sidebar": true}}` which turns on the sidebar, but if you change it to _**false**_ then your rendered document will not have that sidebar. Whatever works best for you.
```

Comment:
Please make here multiline code blocks for `config='{"show": {"sidebar": true}}` :)

Reply:
@magicmatatjahu that is now fixed